assets/css/sensei-course-theme/focus-mode.scss

```diff
@@ -1,92 +1,76 @@
-
 $base: '.sensei-course-theme';
 
-.sensei-course-theme__focus-mode-toggle {
-	.sensei-course-theme--focus-mode & {
-		position: absolute;
-		left: 20px;
-	}
-	padding: 4px;
-	white-space: nowrap;
-	display: flex;
-	flex-direction: row-reverse;
-	align-items: center;
-	gap: 12px;
-	border: none;
-	background: none;
+@media screen and (min-width: $breakpoint ) {
+	.sensei-course-theme__focus-mode-toggle {
+		padding: 4px;
+		border: none;
+		background-color: transparent;
 
-	&__disable {
-		display: none;
-	}
+		&__label {
+			position: absolute;
+			top: -99999px;
 
-	.sensei-course-theme__focus-mode-toggle-icon {
-		width: 10px;
-		height: 10px;
-		transform: rotate(180deg);
-	}
-}
-@media screen and (min-width: 782px) {
-	.sensei-course-theme__sidebar {
-		padding-top: 32px;
+			&--disable {
+				display: none;
+			}
+		}
+
+		.sensei-course-theme__focus-mode-toggle-icon {
+			width: 10px;
+			height: 10px;
+			transform: rotate(270deg);
+		}
 	}
-}
 
-.sensei-course-theme--focus-mode {
-	--focus-header-height: 45px;
+	.sensei-course-theme--focus-mode {
+		--header-height: 45px;
 
-	.sensei-course-theme {
-		&__header {
-			&__container {
-				height: 39px;
-				overflow: hidden;
-			}
-			.sensei-course-theme-course-progress, .sensei-course-theme-prev-next-lesson-text {
-				display: none;
-			}
-			.wp-block-site-logo {
-				margin-right: -12px;
```

Comment:
@yscik , let me know if I missed anything here. I changed it to `display: none` directly to avoid the `gap` and avoid fixing it with the negative margin.

Reply:
I think it was done that way for animation reasons, looks good with your solution!
One bit, for the title, I think we should remove the transition of the font-size, since it's lagging behind. Or just need to adjust the timing/easing?

Reply:
Good catch! It was transitioning the `h2` and the `a`. And weird that it was increasing the duration. But removed from the `a` and it's fixed now. https://github.com/Automattic/sensei/pull/4684/commits/42ccb7c1e5b3635d92c0a4f182411dd51a6232a9